challenges/0-hello-world/go/wallis6n/challenge.go

```diff
@@ -0,0 +1,3 @@
+func main(){
```

Comment:
Boa noite wallis6n! Creio que haja um erro de sintaxe em seu código, pois há a necessidade de um 'package' executável no começo do seu código e seguido de mais uma linha para a importação de uma lib, no caso você utilizou a lib "fmt" para realizar a função Println. Exemplo abaixo do que pode ser acrescido:
```suggestion
package main
import "fmt"
func main(){
```
Caso ainda possua dúvidas, recomendo visitar a documentação da linguagem!

Reply:
Opa, boa noite paulohgs, obrigado pelo toque, na pressa não tinha percebido que tinha esquecido isso (kappa), obrigado pelo review.

Isso foi resolvido no commit 904f200162045ffc3b6e21eed2a30a77dfa5f96e